<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mp3player</class>
 <widget class="QDialog" name="mp3player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(48, 48, 48);</string>
  </property>
  <widget class="QPushButton" name="oynatButon">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>670</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="duraklatButon">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>670</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sarkileriButon">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>670</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="durdurButon">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>670</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sarkigeriButon">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>670</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="sarkiListesi">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>721</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="begenButon">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>260</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(112, 112, 112);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>begen.png</normaloff>begen.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>65</width>
     <height>65</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="sarkiAdi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>760</y>
     <width>611</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);
color: rgb(225, 225, 225);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>(Şarkı Adı)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="sarkiYolu">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="sarkiKonumu">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>630</y>
     <width>721</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 48, 48);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sesSeviyesi">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>620</y>
     <width>31</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(48, 48, 48);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sarkileriSar">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>670</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sarkigeriSar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>670</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);
background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="muteButon">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>750</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sesSeviye">
   <property name="geometry">
    <rect>
     <x>934</x>
     <y>590</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>%50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="cikisButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);
color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>Cikis Yap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="favorilerButon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);
color: rgb(225, 225, 225);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Favorileri Goster</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="begenmeButon">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>370</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>begenme-.png</normaloff>begenme-.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>65</width>
     <height>65</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="InfoLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>790</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 17, 0);
font: 9pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
